### PR TITLE
Add settings for playwright-pytest

### PR DESCRIPTION
**Pull request by keisuke-umezawa**
## Contributor License Agreement

This repository (``optuna-dashboard``) and [Goptuna](https://github.com/c-bata/goptuna) share common code.
This pull request may therefore be ported to Goptuna.
Make sure that you understand the consequences concerning licenses and check the box below if you accept the term before creating this pull request.

* [x] I agree this patch may be ported to [Goptuna](https://github.com/c-bata/goptuna) by other Goptuna contributors.

## Reference Issues/PRs

NA

## What does this implement/fix? Explain your changes.

Add settings for playwright-pytest based on [Installation Guide](https://playwright.dev/python/docs/intro)
The current test is just copied from the installation guide. I will replace it in the following PRs.

### CODE REVIEW

File: .github/workflows/e2e-tests.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - run: npm install
+      - run: npm run build:dev
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
```
Comment (c-bata) [inline .github/workflows/e2e-tests.yml:33]: How about using Python 3.10?

FYI: The reason why visual regression tests uses Python 3.8 is just pyppeteer does not supprt Python 3.9 or later.

**Review by c-bata — COMMENTED** [review]
Looks almost good to me.

File: .github/workflows/e2e-tests.yml (hunk 1 above)
Comment (keisuke-umezawa) [reply .github/workflows/e2e-tests.yml]: Updated.

**Comment by keisuke-umezawa** [comment]
@c-bata 
I updated python version. Could you check it again?

File: .github/workflows/e2e-tests.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+name: e2e-tests
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/integration-tests.yml'
```
Comment (c-bata) [inline .github/workflows/e2e-tests.yml:7]:
```suggestion
      - '.github/workflows/e2e-tests.yml'
```

**Comment by keisuke-umezawa** [comment]
@c-bata 
I merged your suggestion. Could you check it again?

**Review by c-bata — APPROVED** [review]
Thanks for the update! LGTM.

**Pull request merged**